pkg/aggregator/aggregator.go

```diff
@@ -257,3 +267,72 @@ func mergeSpecs(dest, source *spec.Swagger, renameModelConflicts, ignorePathConf
 	}
 	return nil
 }
+
+// deepEqualDefinitionsModuloGVKs compares a and b, but ignores the x-kubernetes-group-version-kind extension.
```

Comment:
can this temporary modification changes the order of fields? that could be ignorable, but I prefer not to mutate the objects we promised we won't even temporarily. what about checking deepEqual on spec.Schema.SchemaProps and spec.Schema.SwaggerSchemaProps and manually on spec.Schema.Extensions?

Reply:
actually we can just set the whole map to nil, do comparison and then compare the old values independently. No modification necessary.

Reply:
fixed